.aicoe-ci.yaml

```diff
@@ -1,15 +1,11 @@
 # Setup and configuring aicoe-ci with configuration file `.aicoe-ci.yaml`
 # Example `.aicoe-ci.yaml` with a full list of config options is available here: https://github.com/AICoE/aicoe-ci/blob/master/docs/.aicoe-ci.yaml
----
-check: []
-# Uncomment following line to build a public image of this repo
-# - thoth-build
-# Uncomment following lines to build a public image of this repo
-# build:
-#   build-stratergy: Source
-#   build-source-script: "image:///opt/app-root/builder"
-#   base-image: quay.io/thoth-station/s2i-custom-notebook:latest
-#   registry: quay.io
-#   registry-org: aicoe
-#   registry-project: <CHANGE-ME>
-#   registry-secret: aicoe-pusher-secret
+check:
+  - thoth-build
+build:
+  build-strategy: Containerfile # Allowed values: Source, Dockerfile, Containerfile (default: Source)
```

Comment:
lets remove the comments.

Reply:
:+1:

Reply:
I would actually be interested in having all these yaml files have LOTS of comments, describing each field, or at least one comment pointing future contributors to full documentation of all fields.